### PR TITLE
Handle buttons task list

### PR DESCRIPTION
**Pull request by Catalin12**

### CODE REVIEW

File: src/app/shared/click-type.enum.ts [hunk 1]
```diff
@@ -0,0 +1,5 @@
+export enum ClickTypeEnum {
```
Comment (danielursu) [inline src/app/shared/click-type.enum.ts]:
```suggestion
export enum FormTypeEnum {
```

File: src/app/shared/click-type.enum.ts [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+	EDIT = "Edit",
+	ADD = "Add",
+	NULL = "Null"
```
Comment (danielursu) [inline src/app/shared/click-type.enum.ts]:
```suggestion
```

File: src/app/task-form/task-form.component.ts [hunk 3]
```diff
@@ -1,32 +1,40 @@
...
 
 @Component({
 	selector: "task-form",
 	templateUrl: "./task-form.component.html",
-	styleUrls: ["./task-form.component.css"]
+	styleUrls: ["./task-form.component.css"],
 })
 export class TaskFormComponent implements OnInit {
 
+	public clickType = this.dialogService;
```
Comment (danielursu) [inline src/app/task-form/task-form.component.ts]: what is clickType?

File: src/app/task/task.component.ts [hunk 4]
```diff
@@ -12,21 +15,44 @@ export class TaskComponent {
 	@Input()
 	public task: TaskDTO = {};
 
+	@Output()
+	public changeStatus = new EventEmitter<number>();
```
Comment (danielursu) [inline src/app/task/task.component.ts]:
```suggestion
	public onChangeStatus = new EventEmitter<number>();
```

File: src/app/task/task.component.ts [hunk 5]
```diff
@@ -12,21 +15,44 @@ export class TaskComponent {
...
+
+	@Output()
+	public clickDelete = new EventEmitter<number>();
```
Comment (danielursu) [inline src/app/task/task.component.ts]:
```suggestion
	public onDelete = new EventEmitter<number>();
```

File: src/app/app.module.ts [hunk 6]
```diff
@@ -9,6 +9,8 @@ import { DialogModule } from "primeng/dialog";
 import { ReactiveFormsModule } from "@angular/forms";
```
Comment (danielursu) [inline src/app/app.module.ts:4]: please resolve conflicts

**Review by danielursu — CHANGES_REQUESTED** [review]

File: src/app/task-form/task-form.component.ts (hunk 3 above)
Comment (Catalin12) [reply src/app/task-form/task-form.component.ts]: from past tests, I forgot to delete it

File: src/app/app.module.ts [hunk 7]
```diff
@@ -42,6 +44,8 @@ import { NavBarComponent } from "./nav-bar/nav-bar.component";
 		ReactiveFormsModule,
 		CardModule,
 		ToggleButtonModule,
+    InputTextModule,
```
Comment (carmina-c) [inline src/app/app.module.ts]: tab here

**Review by carmina-c — APPROVED** [review]

**Pull request merged**